modules/dynamic-plugins/rhdh-supported-plugins.template.adoc

```diff
@@ -0,0 +1,59 @@
+[id="rhdh-supported-plugins"]
+= Preinstalled dynamic plugin descriptions and details
+
+// This page is generated! Do not edit the .adoc file, but instead run rhdh-supported-plugins.sh to regen this page from the latest plugin metadata.
+// cd /path/to/rhdh-documentation; ./modules/dynamic-plugins/rhdh-supported-plugins.sh; ./build/scripts/build.sh; google-chrome titles-generated/main/plugin-rhdh/index.html
+
+[IMPORTANT]
```

Comment:
wasn't sure if you wanted to keep this header in place, or move it down to the tech preview section @Gerry-Forde